dia/diameter_nasreq_rfc7155.dia

```diff
@@ -0,0 +1,341 @@
+Acct-Link-Count                              	51       	Unsigned32          	M   
+NAS-Port-Type                                	61       	Enumerated          	M   
+Port-Limit                                   	62       	Unsigned32          	M   
+Login-LAT-Port                               	63       	OctetString         	M   
+Tunnel-Type                                  	64       	Enumerated          	M   
+Tunnel-Medium-Type                           	65       	Enumerated          	M   
+Tunnel-Client-Endpoint                       	66       	UTF8String          	M   
+Tunnel-Server-Endpoint                       	67       	UTF8String          	M   
+Acct-Tunnel-Connection                       	68       	OctetString         	M   
+Tunnel-Password                              	69       	OctetString         	M   
+Connect-Info                                 	77       	UTF8String          	M   
+Tunnel-Private-Group-Id                      	81       	OctetString         	M   
+Tunnel-Assignment-Id                         	82       	OctetString         	M   
+Tunnel-Preference                            	83       	Unsigned32          	M   
+Acct-Tunnel-Packets-Lost                     	86       	Unsigned32          	M   
+NAS-Port-Id                                  	87       	UTF8String          	M   
+Framed-Pool                                  	88       	OctetString         	M   
+Tunnel-Client-Auth-Id                        	90       	UTF8String          	M   
+Tunnel-Server-Auth-Id                        	91       	UTF8String          	M   
+Originating-Line-Info                        	94       	OctetString         	M   
+NAS-IPv6-Address                             	95       	OctetString         	M   
+Framed-Interface-Id                          	96       	Unsigned64          	M   
+Framed-IPv6-Prefix                           	97       	OctetString         	M   
+Login-IPv6-Host                              	98       	OctetString         	M   
+Framed-IPv6-Route                            	99       	UTF8String          	M   
+Framed-IPv6-Pool                             	100      	OctetString         	M   
+Accounting-Input-Octets                      	363      	Unsigned64          	M   
+Accounting-Output-Octets                     	364      	Unsigned64          	M   
+Accounting-Input-Packets                     	365      	Unsigned64          	M   
+Accounting-Output-Packets                    	366      	Unsigned64          	M   
+NAS-Filter-Rule                              	400      	IPFilterRule        	M   
+Tunneling                                    	401      	Grouped             	M   
+Accounting-Auth-Method                       	406      	Enumerated          	M   
+QoS-Filter-Rule                              	407      	QoSFilterRule       	M   
+Origin-AAA-Protocol                          	408      	Enumerated          	M   
+
+@messages
+AC-Request ::= <Diameter Header: 271, REQ, PXY>
```

Comment:
I called it `AC-Request` like in RFC 7155 [3.9.  Accounting-Request (ACR) Command](https://tools.ietf.org/html/rfc7155#section-3.9). Should I rename to `ACR`?